### PR TITLE
Added guide for setting up a volume

### PR DESCRIPTION
**Pull request by Elden123**
…r /volumes/types

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -968,21 +968,23 @@ components:
...
-          $ref: '#/components/schemas/Cloud'
-        region:
-          $ref: '#/components/schemas/Region'
-        supports_multi_mount:
-          $ref: '#/components/schemas/VolumeSupportsMultiMount'
-        fixed_size:
-          $ref: '#/components/schemas/VolumeFixedSize'
-        price_per_gb_per_hour:
-          $ref: '#/components/schemas/VolumePricePerGBPerHour'
+      type: array
```
Comment (Elden123) [inline openapi.yaml:971]: I noticed that we had this set up to show it returned a single item, it actually returns an array though.

File: guides/attachvolume.mdx [hunk 2]
```diff
@@ -0,0 +1,197 @@
+---
+title: 'Setup a Volume'
+icon: 'database'
+iconType: 'solid'
+---
+
+Volumes provide persistent block storage that can be attached to instances for storing data independently of the instance lifecycle. This makes them ideal for scenarios where you need to maintain critical data even after instances are deleted or replaced. Volumes can be reused and mounted across multiple instances, providing flexibility and durability for your storage needs.
+
+In this guide, we will cover how to create a volume, attach it to an instance, and verify that it is mounted via the Shadeform API. This process involves creating a volume first, launching an instance with the volume attached, and confirming the volume on the instance.
+
```
Comment (cholical) [inline guides/attachvolume.mdx:10]: Can we add a line here that says "Currently, volumes can only be attached to instances in the same cloud provider and region. Multi-cloud volumes are an upcoming feature."

File: guides/attachvolume.mdx [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+Save the `id` field as you will need it in the next step.
+
+---
+
+### Step 3: Create an Instance with the Volume Attached
+
+To attach the volume, pass its `id` in the `volume_ids` field while creating the instance using the [`/instances/create`](https://docs.shadeform.ai/api-reference/instances/instances-create) endpoint. 
+
+<Note> "volume_ids" must be an array of size one. This is because we only allow attaching one volume to one instance right now. Attaching multiple volumes to a single instance is a feature we may add in the future. </Note>
```
Comment (cholical) [inline guides/attachvolume.mdx]: Can we change from "we may add" to "we will add"

File: guides/attachvolume.mdx [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+After connecting, run the following command:
+
+```bash
+# List all block devices attached to the instance
+lsblk
+```
+
+This command will display all block devices attached to the instance. You should see your volume listed as a new device, such as `/dev/sdb` or similar.
+
```
Comment (cholical) [inline guides/attachvolume.mdx:185]: Can we add a new step here that is "Step 5: Mount the volume to a file system" and then show the commands:

sudo mkdir /dev/sdb
sudo mkfs.ext4 /dev/sdb
sudo mount -t ext4 /dev/sdb /mnt/sdb/
sudo chown -R shadeform:shadeform /mnt/sdb/

File: guides/attachvolume.mdx [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+### Step 4: Verify the Volume on the Instance
+
+Once the instance is active, SSH into it to verify that the volume has been mounted. Learn how to SSH into a Shadeform instance using [this guide](https://docs.shadeform.ai/getting-started/quickstart#sshing-into-the-instance).
+
+After connecting, run the following command:
+
+```bash
+# List all block devices attached to the instance
+lsblk
```
Comment (cholical) [inline guides/attachvolume.mdx:148]: Let's also add the example output of lsblk

**Review by cholical — CHANGES_REQUESTED** [review]

File: guides/attachvolume.mdx [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+This command will display all block devices attached to the instance. You should see your volume listed as a new device, such as `vdb`, `sbd`, or similar.
+
+---
+
+### Step 5: Mount the volume to a file system
+
+After verifying the volume is attached, you can mount it to a file system. Find your volume from the `lsblk` command (my volume is called `vdb` in this example) and mount it to a directory using the following commands:
+
+```bash
+sudo mkdir /dev/vdb
```
Comment (cholical) [inline guides/attachvolume.mdx:177]: We don't actually create /dev/vdb. It's not a real folder but a file system representation of a device. This is something that linux does for devices.

**Review by cholical — APPROVED** [review]

**Pull request merged**